src/index.js

```diff
@@ -1,17 +1,18 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { BrowserRouter } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import store from './redux/configureStore';
 import './index.css';
 import App from './App';
-import reportWebVitals from './reportWebVitals';
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <Provider store={store}>
+      <BrowserRouter>
+        <App />
```

Comment:
The app shows a blank page so I am unable to test the functionality and design. Please make sure you are creating a pull request from the branch where the project code resides.
![blank](https://user-images.githubusercontent.com/35504149/163439876-ff5e8584-681f-4dcf-8bea-e09b708f2902.png)

Reply:
This was a result of my wrong implementation of GitHub Pages. Now it works on Netlify.